src/cross_section/src/cross_section.cpp

```diff
@@ -132,6 +132,58 @@ void decompose_hole(const C2::PolyTreeD* outline,
   }
 }
 
+bool v2_lesser(glm::vec2 a, glm::vec2 b) {
+  if (a.x == b.x) return a.y < b.y;
+  return a.x < b.x;
+}
+
+double cross(glm::vec2 a, glm::vec2 b) { return a.x * b.y - a.y * b.x; }
+
+double distance(glm::vec2 a, glm::vec2 b) {
+  auto d = a - b;
+  return std::sqrt(d.x * d.x + d.y * d.y);
+}
+
+bool is_cw(glm::vec2 a, glm::vec2 b, glm::vec2 c) {
+  double lhs = cross(a - c, b - c);
+  double rhs = 1e-9 * distance(a, c) * distance(b, c);
+  return lhs < -rhs;
```

Comment:
If you have a bunch of points that are colinear within precision with their neighbors, but slowly curve convex or concave, which one will get incorrect results? What's the worst case?

Reply:
I'm not totally sure, but this is the more strict version that doesn't include collinear points in the solution, `lhs <= rhs` is supposed to keep them (going by the note in my docs, which was probably based off of BOSL2 since that is what I followed when implementing it in OCaml). The more lax conditional might be more robust to the case that you describe.

Reply:
Anyway, might make for a good test. We had the bug earlier on a high-res sphere where my decimator epic-failed. A high-res circle might be good here, though using doubles will make it pretty hard - maybe just a secant.

Reply:
Got it to fail (a little) for `circularSegments=1e6`.
```
/home/geoff/git/manifold/test/cross_section_test.cpp:189: Failure
Expected equality of these values:
  circ_area
    Which is: 314.15927
  (circ - circ.Scale({0.8, 0.8})).Hull().Area()
    Which is: 314.15906
```